crates/oxc_linter/src/rules/eslint/prefer_spread.rs

```diff
@@ -52,129 +112,173 @@ impl Rule for PreferSpread {
             return;
         };
 
-        let Some(member_expr) = call_expr.callee.without_parentheses().as_member_expression()
-        else {
-            return;
-        };
+        check_eslint_prefer_spread(call_expr, ctx);
+        check_unicorn_prefer_spread(call_expr, ctx);
+    }
+}
 
-        let Some(static_property_name) = member_expr.static_property_name() else {
-            return;
-        };
+fn check_eslint_prefer_spread(call_expr: &CallExpression, ctx: &LintContext) {
+    if !is_method_call(call_expr, None, Some(&["apply"]), Some(2), Some(2)) {
+        return;
+    }
 
-        match static_property_name {
-            // `Array.from()`
-            "from" => {
-                if call_expr.arguments.len() != 1 || member_expr.is_computed() {
-                    return;
-                }
+    let callee = call_expr.callee.without_parentheses();
+    let callee = match callee {
+        match_member_expression!(Expression) => callee.to_member_expression(),
+        Expression::ChainExpression(chain) => match chain.expression {
+            match_member_expression!(ChainElement) => chain.expression.to_member_expression(),
+            ChainElement::CallExpression(_) => return,
+        },
+        _ => return,
+    };
```

Comment:
It may better to move this method to `ast-utils.rs` ?
like `get_callee_as_member_expression(&call_expr)`